### PR TITLE
Add Asylum follow-up 1

### PR DESCRIPTION
**Pull request by YaremenkoM**
### Project Abstract

This application is the follow up to the [Asylum grant application](https://github.com/w3f/Grants-Program/pull/805).

Asylum is a platform for the next generation of gaming, united into the single Metaverse, where anyone has a place to play, study, work and express themselves. Within this application we are building the PoC for the Asylum which includes standards, SDK, and UI apps for our Metaverse ecosystem.

### For which grant [level](https://github.com/w3f/Grants-Program#level_slider-levels) are you applying?

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $50,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for > $100k Web3 Foundation Council approval)

### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [x] I have read and understood the [FAQs](https://github.com/w3f/Grants-Program/blob/master/docs/faq.md), [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/grant_guidelines_per_category.md) and [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/announcement-guidelines.md).
- [x] A BTC, Ethereum (USDT/USDC/DAI) or Polkadot/Kusama (aUSD) address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledge the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process).
- [ ] I prefer the discussion of this application to be in a private Element/Matrix channel. My username is: ``@_______``

### How Did You Hear About our grants program?

- [ ] Social Media
- [ ] Hackathon
- [x] Personal Recommendation - Supercolony
- [ ] Substrate Builders Program
- [ ] Investor/VC
- [ ] Online Search
- [ ] Other: _______

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Sorry for the late reply here. We were partly waiting for the evaluation of your current milestone. I will mark the application as ready for review. 
But I have a few comments/questions: 

- In general, are you already in the process of collecting seed funding and did you already apply for the https://substrate.io/ecosystem/substrate-builders-program/? If not we would be willing to help you with both. To be honest, I have the feeling you are at a stage in which the grants program is no longer the correct tool for you. For example: Milestone 3 — Interoperable games doesn’t necessarily provide a lot of value for other projects in the ecosystem and seems to be more than a poc.   
- What are the features of the Pallet - Template DAO, that are unique and not necessarily handled by some of the other dao pallets?

File: applications/asylum_follow_up_1.md [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+
+#### Asylum overview
+
+The Asylum project is about building the ecosystem, where every сomponent is connected with each other, achieving a synergy.
+
+So, the best way to give a detailed description of the Asylum - is to define its main components:
+
+- **Asylum standards**
+
+In our conception **in-game item** can be represented as an NFT metadata followed by a certain standard. The main goal of the standard is to make every item independent of the concrete game. Storing various possible interpretations will give the ability to use them in different games. On the high level, it means that if two RPG games will use these standards you will be able to buy a single N to buy a single NFT artifact and use it in both games.
```
Comment (semuelle) [inline applications/asylum_follow_up_1.md:33]:
```suggestion
In our conception **in-game item** can be represented as an NFT metadata followed by a certain standard. The main goal of the standard is to make every item independent of the concrete game. Storing various possible interpretations will give the ability to use them in different games. On the high level, it means that if two RPG games will use these standards you will be able to buy a single NFT artifact and use it in both games.
```

File: applications/asylum_follow_up_1.md [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+
+We will build an SDK that wraps ecosystem functionality into a clear interface and simplifies integration. Initially, we will provide two possible options: SDK for game engine (Unity, UE) and agnostic JS connection library.
+
+- **Creator tools**
+
+The initial tool for the Creators is **"Game Developers Console"** - a web application allowing Creators to set up the game in the Game Distribution System, configure in-game items and smoothly test them with different game clients.
+
+- **Asylum App**
+
+In the Asylum App, Players will be able to browse existing games on the platform, buy them, play, manage their inventory, and use the marketplace - everything like on common gaming platforms but fully decentrali platforms but fully decentralized
```
Comment (semuelle) [inline applications/asylum_follow_up_1.md:55]:
```suggestion
In the Asylum App, Players will be able to browse existing games on the platform, buy them, play, manage their inventory, and use the marketplace - everything like on common gaming platforms but fully decentralized.
```

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the follow-up application, @YaremenkoM. I added some questions and suggestions below. 

Overall, what is a bit confusing is that this is almost verbatim your first application. I understand that you are working on the big picture, but - apart from the milestones - we are now reviewing almost the same content. At least the `Development Status` section should be updated since you already submitted two milestones since your first application. And it would be interesting for us to hear what might have changed since your first submission, what you might have learned while working through the first grant, and if anything might have changed about your position or outlook.

**Comment by Noc2** [comment]
I’m closing the application due to inactivity. Let me know if I should reopen it.

**Comment by YaremenkoM** [comment]
Hey @Noc2 @semuelle !

Thank you for your review!

Sorry for being quite inactive within this application, our team was doing lots of job regarding product design and strategy!
We've already started to prepare updates regarding your comments, will rearrange the application this week, and look forward to your review!

**Comment by Noc2** [comment]
In this case, I will reopen the application.

**Comment by YaremenkoM** [comment]
Hi guys!
Thank you for holding our application while we were working on its updates!

Also thanks a lot for your comments and deep look into our application, we appreciate it ❤️

We updated the whole application so now there is our actual vision at the moment, also added an overview of the protocol that we are building. Hope now it looks better!

Answering your questions @noc2:
 ⁃ We’re now preparing for our seed fundraising and see these milestones as the final movement before we start the round. We applied to the Substrate builders program, but not sure if funding from it will cover our needs. That's why we would like to apply to web3 grants if it sounds reasonable for you
 ⁃ I can agree that M3 (Interoperable games) separately can look not valuable for the ecosystem, but it is a part of our platform, which brings that value. These games are needed to create a showcase for people how interoperability can work. For players, it is a point to understand this new gaming experience and so drive this idea; for developers, it is a point to see how such interoperability could be implemented in their games: and finally, for us - it is a way to understand how we can make our SDK usable for real games. We believe that such a showcase will attract more attention and people to this part of the ecosystem and will make it stronger
 ⁃ Template DAO pallet will be used to introduce governance into the NFT evolution and extension. For sure, we will try to reuse existing pallets (like the substrate Governance pallet), but I think there are no existing pallets that are fully covering our needs. For now, we do not have a clear structure of how it will works, as we plan to develop it (structure) in process of milestone delivery, but we have the goals that we want to achieve with it (defined [here](https://github.com/w3f/Grants-Program/pull/1000/files#diff-12ce6a54d50b11e171055376d362f1d9d2bd719be9460e2f6f89ad1a91065182R357))

Thanks for your comment @semuelle!
Yes, you’re right, our applications were very similar to the previous one, it’s just because we thought it should be, looking to another projects applications. Sorry for that, now we have an updated version with a lot of new info

Looking forward to your feedback!

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks a lot for the update and sorry for the late reply here. Could you sign the latest version of our Terms & Conditions? Regarding the funding of this grant, given the updated scope, I’m personally happy to go ahead with it. But just be aware that other committee members might feel differently. However, I have two additional requests: could you change the “Test-net deployment” delivery and only focus on the development work, which you do as part of the delivery. We usually don’t support deployment/hosting and focus only on open source software development.  Furthermore, do you already have some initial mock-ups or designs that you could integrate into the application regarding the front-end?

**Comment by YaremenkoM** [comment]
Thank you for comment!
Latest version of our Terms & Conditions in signed.

Other updates regarding your comment are coming!

**Review by Noc2 — DISMISSED** [review]
Thanks for the update. I’m happy to go ahead with it and I will share it again with the committee.

**Comment by YaremenkoM** [comment]
@semuelle Let us know is there any blockers and looking forward to you review!

**Comment by YaremenkoM** [comment]
Hey @Noc2!

We know that Web3Foundation have a lot of application to review now, but just letting you know that we are glad to go forward with reviewing and ready to help if there are any blockers!

**Comment by Noc2** [comment]
Thanks for chasing us. I will share the application again with the rest of the team.

**Comment by YaremenkoM** [comment]
Any updates here? 😊
Would be really grateful for any estimation for the review🙏

**Comment by cruikshankss** [comment]
Good evening @YaremenkoM,

Sebastian is out of office until October, so please forgive his non-response.

I've been very much looking forward to reviewing your application for over a week and glad to say today I've been able to deeply study your application and the conversation so far on this application. I've also looked over your past grant deliveries, pitch deck, website and litepaper. 

I feel most certainly that I will approve your proposal (just one approval of course). I took very detailed notes in my journal and, well, it's quite late here in the US. Your application is extremely complex; this is a necessity for a true metaverse foundation of course. :) Thank you because you have expressed yourselves clearly, simply & atomically. 

This app deserves a clear head and a couple hours on Monday so I can type out my humble recommendations/feedback in terms of business strategy, software architecture and desire to see a genuine renaissance of storytelling, expression, experience, and art... humanity in the multiverse. :)

Gaming is very near to my heart and oddly, this is the first application which made me well up with tears seeing some of my childhood dreams truly having the right foundation to perhaps become a reality. Cheers and thank you for the inspiration. Please expect a lot of notes on Monday :) oh, yes, but one note: if you don't change your website & other marketing materials from "Have you seen Ready Player One?" to "Have you read/seen Ready Player One?", then I'm afraid I'll never be able to accept your application. ;-)

**Comment by YaremenkoM** [comment]
Hey @cruikshankss !

**Comment by YaremenkoM** [comment]
Hey @cruikshankss !

Thanks for your feedback from all my heart, these words is what makes us move forward!
Already waiting for your notes and very glad to update the texts!

**Comment by 0xMarkian** [comment]
Thanks a lot Ashley @cruikshankss !
Matters a lot to hear

And thanks for noticing our fundamental appproach. I am not sure how much you are familiar with IMP Protocol but I would appreciate if you could look into it !

https://asylumspace.notion.site/IMP-Consortium-paper-baedc7a41c7d4ed290f6cf99b86ceefb

We would like to potentially see you in Consortium!

**Review by cruikshankss — APPROVED** [review]
@YaremenkoM – glad to hear you’re willing to update the texts! And furthermore, super glad to hear how connected your heart is to this project!!

@0xMarkian  – thanks for the link to the IMP Protocol Notion. I had a brief look through, and I don’t want to delay this grant’s review any longer, so I’m not thoroughly reviewing it, although it is a really exciting collection of documents. I’m not sure if it would be a conflict of interest for me to take part in Consortium, and plus right now I’m fully committed to my own work here at w3f and art in my free time, but thank you very sincerely for the invitation! It’s definitely something I’d consider for the future, all factors allowing.

Here are my notes:

Regarding Sebastian’s question regarding a “token swap protocol”, I think it makes a lot of sense to have an exchange. I grew up in the USA going to GameStop stores for my video games and reading their gaming magazines. I know $GME is considered a meme stock, but currently, GameStop is building an NFT marketplace (in beta) (https://nft.gamestop.com/), which I’m currently quite excited about actually, because I think it’s one of the best use-cases and truly valuable business models for NFTs I’ve seen so far. They’ve partnered with a Layer 2 zkRollup on ETH ($LRC) to reduce gas fees and also Immutable X ($IMX) who is building the marketplace I believe. The metaverse is such a massive project that it’s cool to see this collaboration among so many teams, especially since GameStop is connected to so many game studios. Therefore, I think your business plan of an early focus on DeFi elements/games is proper, although I’m not sure if I agree with your “fixed value” idea. I’d have to learn more. By the way, I like that you’re focusing first on Kusama and then Polkadot and also it makes sense to focus on events and social/education later in my opinion.

On that previous point, I’m sure you’d agree that of course developer experience (DX) education/documentation is essential for early adopters, which is why you have a strong focus on your SDK for Unity & Unreal. I’ve never been a game dev so I don’t really know the gaming tech stacks, but I encourage you to stay up to date on the bleeding edge of gaming tech, although Unity & Unreal are what I’ve heard of regarding indie development. I don’t even know, for example, if the Bethesda game engine is proprietary or open-source (I’m a big Elder Scrolls & Fallout fan and I know their engine was quite a meme in the Oblivion/Skyrim days but I’m looking forward to Starfield). I’m just speaking out loud here dreaming about the future, and it’s worth mentioning I’m also very interested in AR/VR gaming tech stacks and how you’ll build a base to accommodate all these different engines/stacks as well as how to connect to platforms like Steam/Battle.net (or diverge from them like Deno is diverging from npm for evolutionary/philosophical reasons).

A big point I’d like to make is: I think it’s ***vitally*** important to focus on pure and genuine high-quality art/games from an early stage, which I hope you’ve already started doing. I find personally it’s simply not worth making games unless you’re working with the either upcoming or established loremasters and artists in the industry. Put another way: I don’t think it’s worth it to make games unless you’re trying to build truly the best games you possibly can–paradigm-shifting, consciousness-shifting art that pushes humanity forward. :)

I love the design idea of your Asylum App. I think it’s a great, classic design idea reminiscent of the Matrix training program/Assassin’s Creed memory corridor/Fallout memory den as well as the metaverse access in some of my favorite sci-fi books including Feed, Uglies/Pretties/Specials, Snowcrash and Ready Player One. 

Regarding your Creator Studio “mods”, I highly suggest you study the YouTube Studio and Patreon, because I think those two are some of the best platforms for artists/creators. Especially Patreon even found a huge niche and is really special; the creator Jack Conte is an artist himself which is really special and speaks deeply about preserving the sacred relationship creators have with their fans. And then in my opinion, YouTube is the best social network on the planet and I really enjoy what Google’s done with the YouTube Studio. 
 
Okay, technically now, I like your idea for NFT interoperability. It reminds me of emoji unicode standards and how different platforms implement different emojis in their own art style. Great idea!!

Regarding your DAO protocol, of course DAOs will be necessary. However, I remember being absolutely heartbroken by the 2016 DAO Ethereum hack. Simply put, I think everything must be done to focus firstly on an extremely robust cryptographically secure foundation before building DAOs. Identity and DAOs are like the holy grail of crypto in my opinion, so please be extremely careful to prevent the suffering of people by taking great care with building DAOs. That being said, I like your DAO-within-a-DAO-within-in-DAO layering model. ;-) Very smart!! I’d love to riff on DAOs and identity going forward with you in the future, because it’s something I think about quite a lot both technically and philosophically. I think NFTs and intellectual property are only ***truly*** solved if identity is solved. ;-) I’ll save some of those ideas for later however. ;-) I’m glad you’re focusing on infra, standards and protocols. It’s very wise. Like I mentioned above, I trust you to hand the IMP and will refrain from my technical comments on that for the time being. It’s just a lot for me to process right now and I hope these comments are enough at this stage. :)

One humble suggestion: you may want to consider renaming your NFT “templates” to “blueprints” because I think it sounds cooler and would excite artists a lot more to feel like digital inventors who are uploading their metaverse blueprints. :) That being said, the logical model of the template/interpretation/tags with the creator and issuer, etc., is ***very*** smart to accommodate artists business models such as licensing and royalties. I think probably it will also be very important for you to evolve business contracts/terms and conditions/legal contracts, etc. with smart contracts for web3 artists to easily do business in the metaverse! 

Regarding TEEs, it’s a complicated topic, and I know a team who may have a better and simpler, extremely clever alternative to TEEs. They’ve completed a w3f grant by the way. If you’re interested in connecting with them, I will speak to them and see if they’re interested in connecting with you! That being said, TEEs may be necessary, but there are so many ideals to conceptualize and achieve in order to truly build them!

By the way, I think it’s really smart that you’re thinking about Layer 2 backends. Layer 2s will almost certainly need a blend of web 2.0 and 3.0 backends for both speed and transparency and I think it will be really cool to see how you can provide guidance for them to evolve step-by-step from web 2.0 to ideal web 3.0 backend tech stacks! The way I think about my purpose as a software architect in general is to help evolve existing tech to higher and higher standards, step by step by building/rebuilding the proper foundations, and eventually hopefully reach all ideals! It’s so great to recognize the value in what’s already built in this world and to work together to reuse/recycle as much software engineering work as possible simply by helping it evolve! I hope you are able to make collaborations with the projects in the ecosystem you’ve mentioned. I have spoken to the Ajuna network team before; I’m extremely excited by what they’re doing and think it would be an excellent non-competitive complement to your work; and happy to ask them if I could make an introduction if you’re interested! No worries if that would make sense at a later stage, however. Also, I see a potential collaboration with RMRK/NFT 2.0 is part of the future OpenBrush plans, so I’m excited to see how this all evolves!

Regarding your world protocol, it totally makes sense that you’ll have to solve governance and economy once you’ve solved DAO security and identity. I’m glad you’re thinking into the future. It’s going to be extremely fascinating to see how to meta-map all the ideals of power dynamics/governance/economics from history and across the world and how to go step-by-step in intermediate stable forms. So many times in history, people’s savings/established value was lost due to inflation due to the monetary system cycles of money printing (Ray Dalio speaks at length on this https://www.youtube.com/watch?v=PHe0bXAIuk0). It’s going to be extremely complex to manage all of this while ***also*** managing the evolvability and upgradability of the underlying technologies at the same time. XD But I reviewed the OpenBrush upgradeable contracts milestone and I think if there’s any collective of engineers who can do this, this is it. :) 

Therefore, with all that being said, I approve this application (one approval)! Lol. Good luck with your _étalons_!

**Comment by YaremenkoM** [comment]
@cruikshankss Thank you a lot for your comment!
We are impressed by your vision and have already begun to parse in detail each part of your message

Will write our detailed feedback asap

**Comment by LastSymbol0** [comment]
Hi @cruikshankss!

First of all, I want to say a big thanks to you for so detailed review and feedback on our application - it is very useful for us to see all these points to improve!

And also I'm very glad to get such feedback from a person with common interests! I'm also a big fan of Elder Scrolls and obviously Ready Player One should be firstly read, not watched 🙂 

Back to your comments:

From what I see we didn't show this concept of "fixed value" of digital objects properly.
So, firstly the problem that we aim to solve with it is - the "in-game" balance between different spaces. If the client app of every space will have the ability to decide the "value" of every object by itself, or if it will be added to an object in the same way as the visuals - we will have a disbalance between spaces. That is the best case will lead to the misconception (Why my super-rare-dragon-sword from Skyrim has only 2 points of damage in Minecraft?) or in the worth case to abusing the system (create a dummy game -> mint items super cheap -> adapt them to use as valuable assets in another game -> sell)
To avoid this problem the best approach that we found so far - add a "value" property to every object, that will be used by every space as a "coefficient" of item evaluability. And this property can be increased only by paying Asylum tokens.
And from this point, we achieve the value that could not be forged and every space could use it as a starting point to define object in-game characteristics.
The "fixed value" in that case means that every object will have its value fixed between spaces.
Will appreciate your feedback on that idea!

Speaking about VR and AR - this is definitely what we have on our radars, but not started investigating yet. I believe it will be the next big point after we'll finish the adaptation to the classical engines.

On the point of platforms like Steam - from what we see for now they are mostly against web3, and it looks more like diverging in the case, as the next evolutionary step for the industry.

Totally agree with you that it only makes sense to focus on creating really high-quality games, not templated ones. The point here is that we are not focused on making games by ourselves (so far 🙂), so now we are looking for the studios to use our ecosystem, that are making such games. It is an important point in our strategy (and the same for non-gaming spaces). Btw, if you have connections with such studios - we will be highly appreciative if you could share them!

Thanks for the feedback on the app design!

Regarding Creator Studio - we will definitely investigate YouTube Studio and Patreon and will see what we could learn from it, thanks for the suggestion!

The structure of DAO and Identity is what we currently actively discussing within a team, so thanks for your attention and warnings on these topics, we will consider it. It will be definitely cool to discuss it when we will be ready with a more detailed and structured vision!

We will consider team "blueprints" as a candidate for renaming, I personally feel as it will be more clear!
Because for now, it could be some mess with naming as building the new concepts it's not so easy to find the most suitable term from the first try, so we are open to such changes.
And yeah, we also see it as a space for more smooth collaboration between artists, developers, and maybe even game designers; give a possibility to monetize their efforts more fair and clear. But so far we didn't consider some legal support for this process, I will put an investigation of this topic in our plans.

We will be definitely interested in connecting with this team you mentioned, who are doing alternatives for TEE. Consideration of solutions in this direction will be important for us to choose the most suitable one. Thanks for the proposal!

I share your thoughts about our mission as architects, about helping evolve and reusing existing code blocks! We're already in conversations both with Ajuna and RMRK teams looking for how we could collaborate with our products to grow this sphere together!


Yes, World protocol seems to be a challenging task in the future, but for me in the meantime, it looks very exciting to work with. Watched the video you shared, very clear explanation!

Thanks again for your feedback and approval on this application!
We're looking forward to working on delivery!

**Comment by YaremenkoM** [comment]
I am glad to go forward with all provided feedback!

@Noc2 @cruikshankss Would be extremely happy to make next steps with application reviewing!

**Comment by cruikshankss** [comment]
Dear @LastSymbol0 & @YaremenkoM,

Thank you so much for your messages. With this info, I'm ready to share your application with the committee again. I've just finished the sol2!nk evaluation, so I have some time to get back to Asylum now. :) 

I'm pretty tired right now but my plan is to share your app on Monday morning and update you before my end of day on Monday (GMT-7:00). 

Hope you have a great weekend!

**Comment by YaremenkoM** [comment]
@cruikshankss Thanks!

It is very important for us to move as fast as possible with this milestones, so looking forward to last review!

**Comment by cruikshankss** [comment]
@YaremenkoM I just discussed your application again with the team and its been shared with the committee again. :)

**Review by alxs — CHANGES_REQUESTED** [review]
Hey @YaremenkoM, thanks for the application. Could you be more precise in describing the deliverables? The specification should be a technical description of the functionality each deliverable provides. Currently, most of them are only vaguely described, such as "expand Unity SDK to make it more abstract", "3D game, which operates 3D NFT representation", "anti-cheat system", " back-end that will track user actions and make decision about items minting" etc. You can think of these as the requirements of a contract. We should be able to assess whether your delivery meets the specification based on them. Here is for example Ajuna's last application, who also developed a Unity SDK among other things: https://github.com/w3f/Grants-Program/blob/master/applications/ajuna_network_follow_up.md

**Comment by YaremenkoM** [comment]
Hey @alxs !

Thanks for your review, making deliverables more specific really makes a lot of sense!

So we extended description of our deliverables and hope it looks more clear now!
Also I want to note that we have slightly revised the human resources we use for these milestones, so we updated milestones' FTE and costs were increased a bit.

Looking forward for your review and approve!

**Review by alxs — APPROVED** [review]
Thanks @YaremenkoM. LGTM.

**Review by Noc2 — APPROVED** [review]

**Comment by YaremenkoM** [comment]
Thanks, guys!

@cruikshankss and your last approval will make us incredibly happy! 
❤️

**Comment by YaremenkoM** [comment]
@cruikshankss Just a kindly reminder that we anticipate your approval! 😊🙏

**Comment by YaremenkoM** [comment]
Hey guys! @Noc2 @laboon @cruikshankss @alxs 

Just wanted to update you regarding our development flow.

We’ve been breathlessly waiting for the approval of our follow-up application to begin the development of 3rd and 4th milestones.

These milestones are very important for our roadmap, so as the approval process took a bit longer than we expected, to not delay the development process and go by our timeline, we have started the development process a bit earlier than the application was submitted.
And according to our forecasts, we’ll be able to deliver the first milestone in a bit less than 2 weeks (as it was initially estimated) and with a dear soul, we’ll ask you to check it. 

Is that timeline ok for you or does the estimated delivery time need to be very strict?

**Comment by alxs** [comment]
@YaremenkoM that's certainly not an issue. Thanks for asking.

**Comment by keeganquigley** [comment]
Hey @YaremenkoM , any updates on how Milestone 4 is coming along?

**Comment by keeganquigley** [comment]
Hi @YaremenkoM just bumping a friendly reminder. Please note that if we don't hear back from you within 2 weeks we will file a PR to terminate the grant due to lack of activity. Thanks!

**Comment by LastSymbol0** [comment]
Hi @keeganquigley !

My apologies for our silence here for such a long time

Unfortunately, we won't be able to deliver the last milestone, as Asylum as a company will be closed

I'm really sorry for that and want to sincerely say thanks to the W3F and all the evaluators supporting us within this program, it was really valuable for us during this path. I hope that what we started to do for games interoperability in Polkadot with Asylum will be the basis for future work in this direction

Please, let me know if there are any extra actions we should do

Thanks!

**Comment by keeganquigley** [comment]
Thanks for letting us know @LastSymbol0 and please do let us know if anything changes. I will go ahead and file a PR to terminate the grant.

We would like to thank you for your interest in the grants program and for your contributions to the ecosystem. You're also welcome to apply for further grants in the future, though please note that the committee will likely take the current termination into account.

**Comment by keeganquigley** [comment]
@YaremenkoM Can you also confirm this to be the case, since you were the original author of the [grant application](https://github.com/w3f/Grants-Program/pull/1000)? Thanks!

**Pull request merged**